### DISCUSSION TITLE
I really like the notification system

### DISCUSSION THREAD

**Discussion by TruePadawan** [mantinedev/mantine#4614] [Kind words]
I randomly came across this library while trying to setup a rich text editor in my project (I'm now using the RichTextEditor component) and had never heard of it previously but I decided to checkout its other components, I saw the notification system and it's so easy and convenient to use. I had been using MUI in the project and was using a Snackbar + Alert component to do notifications but with Mantine I just need to declaratively decide when I want a notification to show and that is SUPER useful for me.
I ended up porting the app from MUI to Mantine entirely, I will be using this from now on, this is a neat component library.

![image](https://github.com/mantinedev/mantine/assets/71678062/95158383-64b5-4637-bc5a-a905ee92ee84)

**Comment by rtivital** [comment]
Thanks for your feedback and kind words!
